### PR TITLE
Gh pages korean

### PR DESCRIPTION
**Pull request by liks79**
# Korean translation of choosealicense.com

Hello.
I think the contents of 'choosealicense.com' are very simple and easy.
It is wonderful!
So, I translated choosealicense.com in Korean.
I made a new branch "gh-pages-korean" for Korean translation.
I've finished translation.

I hope choosalicense.com support Korean.

Please review my work and give me some information how to merge master branch or your plan about support multi-lingual.

Thanks.

### CODE REVIEW

**Comment by liks79** [comment]
If you want to see translation result, visit following url.

http://liks79.github.io/choosealicense.com

File: licenses/agpl.txt [hunk 1]
```diff
@@ -19,7 +21,7 @@ permitted:
   - distribution
 
 forbidden:
-  - no-liability
+  - no-liabilityThe Free Software Foundation recommends taking the additional step of adding a boilerplate notice to the top of each file. The boilerplate can be found at the end of the license.
```
Comment (weakish) [inline licenses/agpl.txt]: Why change this line?

Comment (liks79) [inline licenses/agpl.txt]: Oh,... I'm Sorry.. 
It's my mistake!
I'll be remove it!

Thanks

**Comment by benbalter** [comment]
Awesome, awesome stuff. Would love to figure out the best way to integrate this going forward (#68).

File: _includes/sidebar.html [hunk 2]
```diff
@@ -1,26 +1,40 @@
 <div class='sidebar'>
 
-  <a href="#" data-clipboard-target="license-text" class="js-clipboard-button button">Copy license text to clipboard</a>
+  <a href="#" data-clipboard-target="license-text" class="js-clipboard-button button">라이선스를 클립보드에 복사</a>
```
Comment (weakish) [inline _includes/sidebar.html:3]: Once this button is clicked, the text of the button becomes to 'Copied!' (which is not translated).

I guess the following line in `app.coffee` need translatation:

``` coffee
@innerText = "Copied!"
```

Comment (liks79) [inline _includes/sidebar.html:3]: Yeahh!
That's right.
I'll apply it.

Thanks!

**Comment by haacked** [comment]
Closing this. This would simply translate our existing site to Korean and overwrite the English! We need a better approach for translations. I opened an issue here: https://github.com/github/choosealicense.com/issues/112

**Pull request closed without merging**